modules/Microsoft.Network/virtualNetworks/.test/common/deploy.test.bicep

```
@@ -0,0 +1,128 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for a testing purposes.')
```

Something to update in the utility

```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```